main.tf

```diff
@@ -393,6 +393,8 @@ module "gke" {
     },
   ]
 
+  cluster_resource_labels = var.gke_cluster_resource_labels
```

Comment:
We could use predefined value or maybe a custom value in addiction of a predefined value

Reply:
@Syphon83 do you think we shuold set a default value here ? maybe using the project name ?